libs/api/__tests__/specs/user.test.ts

```diff
@@ -48,185 +52,245 @@ describe('Users', () => {
       ids.unshift(res.data?.createUser.id)
     })
 
-    test('can be created', async () => {
-      const input: UserInput = {
-        name: 'Robin Wayne',
-        email: `${generateRandomString()}@wepublish.ch`,
-        emailVerifiedAt: new Date().toISOString(),
-        properties: [],
-        active: true,
-        roleIDs: []
-      }
+    describe('public', () => {
+      describe('updatePaymentProviderCustomers', () => {
+        test('can add payment provider customers', async () => {
+          const addPaymentProviderCustomers = await testServerPublic.executeOperation({
+            query: UpdatePaymentProviderCustomers,
+            variables: {
+              customers: [
+                {
+                  paymentProviderID: '123-123',
+                  customerID: '1234-1234'
+                },
+                {
+                  paymentProviderID: '122-122',
+                  customerID: '1233-1233'
+                }
+              ]
+            }
+          })
 
-      const res = await testServerPrivate.executeOperation({
-        query: CreateUser,
-        variables: {
-          input,
-          password: 'pwd123'
-        }
-      })
+          expect(addPaymentProviderCustomers).toMatchSnapshot()
+        })
 
-      expect(res).toMatchSnapshot({
-        data: {
-          createUser: {
-            id: expect.any(String),
-            email: expect.any(String),
-            emailVerifiedAt: expect.any(Date)
-          }
-        }
-      })
-      ids.unshift(res.data?.createUser.id)
-    })
+        test('can update payment provider customers', async () => {
```

Comment:
This test makes sure we are not running into the same issues as with the authorIds. This makes sure we can update an already existing value